helm/cluster-azure/values.yaml

```diff
@@ -94,10 +96,10 @@ cluster:
         version: v1beta1
       infrastructureMachinePool:
         group: infrastructure.cluster.x-k8s.io
-        kind: AzureMachineDeployment
+        kind: AzureMachineTemplate
         version: v1beta1
       machineHealthCheckResourceEnabled: false
-      machinePoolResourcesEnabled: false
+      machinePoolResourcesEnabled: true
```

Comment:
What does it mean? Will we use Azure Machine Pools instead of deployments?

Reply:
No we use `AzureMachineDeployment` but we have it behind this flag to enable it from cluster chart.

Reply:
But I remember we want to align in the next months trying to integrate Azure Machine Pools to align with CAPA